src/frontend/components/SøknadsSteg/EøsSteg/Barn/useEøsForBarn.tsx

```diff
@@ -220,6 +222,29 @@ export const useEøsForBarn = (
         nullstillVedAvhengighetEndring: true,
     });
 
+    /*--- BARNETS ADRESSE ---*/
+    const barnetsAdresseVetIkke = useFelt<ESvar>({
+        verdi: formaterVerdiForCheckbox(gjeldendeBarn[barnDataKeySpørsmål.barnetsAdresse].svar),
+        feltId: EøsBarnSpørsmålId.barnetsAdresseVetIkke,
+        skalFeltetVises: () =>
+            (borMedAndreForelder.verdi === ESvar.JA &&
+                skalSkjuleAndreForelderFelt(gjeldendeBarn)) ||
+            gjeldendeBarn.erFosterbarn.svar === ESvar.JA,
```

Comment:
skalSkjuleAndreForelderFelt sjekker allerede om man er fosterbarn

Reply:
Tror man må ha en sjekk for begge tilfeller siden om man er fosterbarn vil man ikke få opp spørsmålet om borMedAndreForelder, men hvis man den andre foreldren er ukjent trenger man også at man svarer ja på borMedAndreForelder

Reply:
Oki, skjønner 👍